salt/utils/__init__.py

```diff
@@ -1732,6 +1732,106 @@ def appendproctitle(name):
         setproctitle.setproctitle(setproctitle.getproctitle() + ' ' + name)
 
 
+def chugid(runas, group=None):
```

Comment:
It looks like these functions were added separately in a different PR and already exist in develop. However, they have been moved to `salt/utils/user.py`. Please check the functions you are adding here against their counterparts in that file and make any changes there. We should not be adding new functions to `salt/utils/__init__.py`.

Reply:
Actually, they already existed when I originally submitted the PR but have since been moved. That was the original reason for needing to rebase after the initial approval, so will be sure to fix that up.